test/specs/commands/createTransactionCreateMultisignatureAccount.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { setUpInputStubs } from '../../steps/utils';
+import * as given from '../../steps/1_given';
+import * as when from '../../steps/2_when';
+import * as then from '../../steps/3_then';
+
+describe('create transaction create multisignature account', () => {
+	beforeEach(() => {
+		setUpInputStubs();
+	});
+	describe('Given a Vorpal instance with a UI and an active command that can prompt', () => {
+		beforeEach(given.aVorpalInstanceWithAUIAndAnActiveCommandThatCanPrompt);
+		describe('Given an action "create transaction create multisignature account"', () => {
+			beforeEach(given.anAction);
+			describe('Given a passphrase "minute omit local rare sword knee banner pair rib museum shadow juice"', () => {
+				beforeEach(given.aPassphrase);
+				describe('Given a Lisk object that can create transactions', () => {
+					beforeEach(given.aLiskObjectThatCanCreateTransactions);
+					describe('Given a keysgroup with keys "+215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca" "+922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa"', () => {
```

Comment:
Missing "and"

Reply:
I prefer that it can be one or many keys with this Given step

Reply:
This is a reason to keep the "and" out of the function name, but not the description which should be as close to natural English as possible.